assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -237,27 +233,30 @@ export default function UserInputQuestionnaire() {
 		<div>
 			{ settingsProgress }
 
-			{ activeSlugIndex <= steps.indexOf( USER_INPUT_QUESTION_ROLE ) && (
+			{ activeSlugIndex <=
+				steps.indexOf( USER_INPUT_QUESTIONS_PURPOSE ) && (
 				<UserInputQuestionWrapper
-					slug={ USER_INPUT_QUESTION_ROLE }
-					isActive={ activeSlug === USER_INPUT_QUESTION_ROLE }
+					slug={ USER_INPUT_QUESTIONS_PURPOSE }
+					isActive={ activeSlug === USER_INPUT_QUESTIONS_PURPOSE }
 					questionNumber={ 1 }
 					title={ __(
-						'Which best describes your team/role in relation to this site?',
+						'What is the main purpose of this site?',
 						'google-site-kit'
 					) }
 					description={ __(
-						'This will help Site Kit show tips that help you specifically in your role',
+						'Based on your answer, Site Kit will tailor the metrics you see on your dashboard to help you track how close you’re getting to your specific goals',
 						'google-site-kit'
 					) }
 					next={ nextCallback }
 					nextLabel={ nextLabel }
+					back={ backCallback }
 					error={ error }
 				>
 					<UserInputSelectOptions
-						isActive={ activeSlug === USER_INPUT_QUESTION_ROLE }
-						slug={ USER_INPUT_QUESTION_ROLE }
-						options={ USER_INPUT_ANSWERS_ROLE }
+						isActive={ activeSlug === USER_INPUT_QUESTIONS_PURPOSE }
+						slug={ USER_INPUT_QUESTIONS_PURPOSE }
+						max={ 2 }
```

Comment:
As can be seen in the Figma designs, the `PURPOSE` question only accepts one answer. Hence, this `max` prop should be removed.